### ISSUE TITLE
README still refers to CloudFormation templates

### ISSUE THREAD

**Issue by fghaas** [openedx-unsupported/configuration#2535]
`README.md`, and the configuration Wiki that it references, still talk about CloudFormation all over the place. Since the CFN templates were unceremoniously killed in 0e2fad115bfe98b665a0f3ccbe26c254f646ce1c being determined "out of date", it would be wise to remove the corresponding references in the documentation as well, so as not to serve as a red herring getting users on the wrong track.

**Comment by andreabedini** [comment]
ah! that explains why I could not find the templates in the repo 😞 ...

**Comment by nicholas-itpedia** [comment]
This needs to be changed. I will try to work on it...

**Comment by plumdog** [comment]
There is a note about CloudFormation being out of date, but I can find no documentation about installing anything other than an all-in-one-server installation. In fact, [this comment](https://openedx.atlassian.net/wiki/spaces/OpenOPS/pages/146440579/Native+Open+edX+Ubuntu+16.04+64+bit+Installation?focusedCommentId=789840338#comment-789840338) seems pretty open there being no help for doing so, and that's just the way it is.

The CloudFormation definitely did this though - the commit linked in the issue description has files like `ElastiCache.json` and `RDS_MySQL_55_With_Tags.json`. These seem to me to be - in principle - amazing resources for anyone wanting to deploy Open edX on AWS.

I know it was an awful long time ago, so many apologies, but perhaps could @fredsmith (who made https://github.com/edx/configuration/commit/0e2fad115bfe98b665a0f3ccbe26c254f646ce1c) or @feanil (who reviewed it at https://github.com/edx/configuration/pull/1984) give some clues about what was out-of-date about the CloudFormation and what might be required to reinstate them or replace them? Is there an associated ticket somewhere for that PR? I'd love to understand what the plan is here.

If, however, they were retired more for "policy" reasons than for technical ones, it would be great to have that made explicit so I don't spend time down the wrong path.

**Comment by fredsmith** [comment]
@plumdog  We (edx.org) stopped using cloudformation in 2014 because it wasn't appropriate for an evolving infrastructure.  Cloudformation templates are complex and don't really allow changes to be captured or reviewed. Since the files were no longer relevant to our infrastructure and not maintained, we pulled them because they weren't an accurate way of deploying openedx, and no longer reflected edx.org.

These days, we deploy edx.org via a combination of Terraform, GoCD, Docker, Ansible and Asgard. Almost all of the tooling we use is open source, but our exact configuration and a start to finish how-to has escaped us due to lack of resources and prioritization.  We also worked with the open source deployer community and determined that a very prescriptive deployment solution wasn't what the community wanted - many people had rolled their own deployment workflow and found flexibility in tooling to be more useful.

I'm sorry there's still no getting started guide for a large deployment of openedx.  We're continuing to work on standardizing our deployments and tooling with simplicity in mind.
